test/src/test-cppapi-aggregates.cc

```diff
@@ -1624,6 +1624,31 @@ TEMPLATE_LIST_TEST_CASE_METHOD(
   array.close();
 }
 
+typedef tuple<std::byte> SumBlobTypeUnderTest;
```

Comment:
```suggestion
typedef tuple<std::byte> BlobTypeUnderTest;
```

Reply:
Note that we are not testing your new types here... Let's file a follow up story for this as it's not trivial.